### DISCUSSION TITLE
Is there support for "await using" or "using" in typescript transform?

### DISCUSSION THREAD

**Discussion by DavidBal** [oxc-project/oxc#7699] [Q&A] [answered]
Hi,

i want to use oxc transform to transform a typescript into javascript.

Question:
Is there support for "await using" or "using" in typescript transform? 

Here is a quick example with using:
https://playground.oxc.rs/#eNpFT81ugzAMfhXLp1ZC7cSRaadNfYEexw4hGBopcVAc2lUo774ERHey/f3ZXlBjg7MYHiGSRPgApgdcvaN4y+Dh+N5yy9oqkX8UjJssOeIo8GVk8qI6Sy0vLQN8X5+u8/bUrwT9HI4N3L3pYWUBzufNQvDpe4IbBYLCpJYTVuixWTDMXIo8OapfbGKYqUJrOO69aD/Ra1gX7lMMimXwwW1AqnBSQSisgX4Omi7GEiuX/Vh+PkXBrCrxqyr3r4zNFlUYKe9Gkvqtrota59tHKlfmwRk2g9nN2nMM3l6sfxTznUKXv8VmUFYopfQHn/OCXQ==

Kind regards,
David

**Comment by camc314** [comment, marked as answer]
added in https://github.com/oxc-project/oxc/pull/9310
